cmd/gcsim/main.go

```diff
@@ -53,6 +55,8 @@ func main() {
 	flag.StringVar(&opt.config, "c", "config.txt", "which profile to use")
 	flag.StringVar(&opt.out, "out", "", "output result to file? supply file path (otherwise empty string for disabled). default disabled")
 	flag.StringVar(&opt.sample, "sample", "", "create sample result. supply file path (otherwise empty string for disabled). default disabled")
+	flag.StringVar(&opt.sampleMinDps, "sampleMinDps", "", "create sample result for the max-DPS run. supply file path (otherwise empty string for disabled). default disabled")
```

Comment:
contains typo, should say min instead of max

Reply:
ah nice catch, fixed